.env.fake

```diff
@@ -6,3 +6,5 @@ APISERVER_AUTH_JWKS_ISS=fake
 APISERVER_AUTH_JWKS_URL=http://localhost:7171/jwk
 
 APISERVER_MARKETS_ADDRESS=fake
+
+REDIS_HOST=redis-stack-apiserver
```

Comment:
As discussed let's try to remove this. If you need the env var you can set in in your Docker command or Docker container.

Reply:
https://github.com/uvio-network/apiserver/pull/49/commits/18f6988212147a8c7d242cb01e64875232c2eda9